virttest/virsh.py

```diff
@@ -30,13 +30,16 @@
 import select
 import locale
 import base64
+import inspect
 
 import aexpect
 from aexpect import remote
 
 from avocado.utils import path
 from avocado.utils import process
 
+from functools import wraps
```

Comment:
this is system packages, please arrange imports by orders: system package,thirty party,then local .

Reply:
Nice catch! updated